README.rst

```diff
@@ -34,9 +34,9 @@ The aws-cli package works on Python versions:
 -------
 Notices
 -------
-On 2024-06-20, support for macOS versions 10.14 and prior was dropped.
-To use up-to-date versions of AWS CLI v2, customers using macOS 10.14 or prior
-should upgrade to macOS 10.15 or later. For more information, please see
+On 2024-11-11, support for macOS version 10.15 was dropped.
```

Comment:
high-level question; do we record the last version that had support (or the version that dropped support) anywhere in our docs? That will probably ease any searching customers have to do down the line.

Reply:
Ah yeah that would be useful. We don't, but I'll talk to someone who can help with that.